crates/typst/src/foundations/scope.rs

```diff
@@ -137,6 +137,17 @@ fn unknown_variable_math(var: &str, in_global: bool) -> HintedString {
         res.hint(eco_format!(
             "`{var}` is not available directly in math, try adding a hash before it: `#{var}`",
         ));
+    } else {
+        res.hint(eco_format!(
+            "if you meant to display multiple letters as is, try adding spaces between each letter: `{}`",
+            var.chars()
+                .flat_map(|c| std::iter::once(' ').chain(std::iter::once(c)))
+                .skip(1)
```

Comment:
intersperse stable when :/
```suggestion
                .flat_map(|c| [' ', c]))
                .skip(1)
```

Reply:
It'll require at least another 3 years of bikeshedding